README.md

```diff
@@ -86,143 +72,25 @@ Please note that this project is released with a [Contributor Code of Conduct](C
 
 ## Development
 
-### Prerequisites
+For development, you'll need [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/) installed.
 
-- Make sure you have Docker and Docker-compose installed in your local environment. For instructions on how to install it, follow the links below:
-  - https://docs.docker.com/get-docker/
-  - https://docs.docker.com/compose/install/
-
-- Environment Variables
-  - Refer to [the example file](.config/.env.example) for the required environment variables.
-
-### Setup and Development Commands
-
-#### To set up and run database, backend (server), and frontend (client) applications:
+To set up your development environment:
 - Run `make local-build` within the root folder of the project to build the application
 - Run `make local-run` within the root folder of the project to run the application
-- You should see the application running at `localhost:4000`
-
-#### To seed the database with data in an Excel sheet:
-
-- `make seed-sites SHEET=my_spreadsheet.xlsx`
-- `make seed-participants SHEET=my_spreadsheet.xlsx`
-
-**Note:** The Excel file should be placed in `server/scripts/xlsx`.
-
-#### To tear down your environment:
-
-- Run `make local-close`
-- Run `make local-clean`
-
-#### To create KC test users as well as assign roles:
-
-- Run `make local-kc-build`
-- Run `make local-kc-run`
-- Run `make kc-import-users`
-Go to `http://keycloak.local.freshworks.club:8080/auth` console and add users/assign roles under the client `hcap-fe-local`
-- Run `make kc-export-users` to save your changes
-`409 Conflict` errors are ok if the user already exists, only the role change will be applied
 
-#### To run server unit tests:
+You should see the application running at `localhost:4000`.
```

Comment:
Worth making this `http://hcapemployers.local.freshworks.club:4000` at this point?

Reply:
Good point. I'll add it to the ticket I am writing for the new setup instructions (HCAP-1378).